### PR TITLE
Corrected Kms key variable name to signify arn instead of id. Removed unused variable availability_zone.

### PR DESCRIPTION
**Pull request by arpit9508**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -55,7 +55,7 @@ resource "aws_db_instance" "this" {
   storage_type      = "${var.storage_type}"
   iops              = "${var.iops}"
   storage_encrypted = "${var.storage_encrypted}"
-  kms_key_id        = "${var.kms_key_id}"
+  kms_key_id        = "${var.kms_key_arn}"
```
Comment (rafikurnia) [inline main.tf]: Yeah you are right, according to https://www.terraform.io/docs/providers/aws/r/db_instance.html 
> kms_key_id - (Optional) The ARN for the KMS encryption key. If creating an encrypted replica, set this to the destination KMS ARN.

File: variables.tf [hunk 2]
```diff
@@ -70,7 +70,7 @@ variable "storage_encrypted" {
   default     = "true"
 }
 
-variable "kms_key_id" {
+variable "kms_key_arn" {
```
Comment (rafikurnia) [inline variables.tf]: Renaming the variable might be better to avoid misleading information

File: variables.tf [hunk 3]
```diff
@@ -92,12 +92,6 @@ variable "parameter_group_name" {
   description = "Name of the DB parameter group to associate"
 }
 
-variable "availability_zone" {
```
Comment (rafikurnia) [inline variables.tf]: But what is the reason you remove this variable? @arpit9508

**Comment by sumodirjo** [comment]
Closing this PR since no responses from the requester and not following guidelines.

**Pull request closed without merging**